userguide/intro.rst

```diff
@@ -493,6 +493,11 @@ U2
 U3
 ~~
 
+* When compatible hardware is connected, the :guilabel:`Multipaths` page
```

Comment:
I'm a little confused with this sentence. It seems to be a typo. Or perhaps 2 sentences?
From what I understand, it appears that the 'Multipaths' page has been added in U3, but it only appears when compatible hardware is connected. So, perhaps:

`A :guilabel:`Multipaths`page now appears in the
:menuselection:`Storage`
menu when compatible hardware is connected.`

Reply:
I tried to put the caveat about compatible hardware first, but I'll rework it. Thanks!